pkg/api/models/availability_zone.go

```diff
@@ -17,6 +17,9 @@ type AvailabilityZone struct {
 
 	// name
 	Name string `json:"name,omitempty"`
+
+	// state
+	State bool `json:"state,omitempty"`
```

Comment:
What does `State: true` mean? We should either have this a string field carrying the state name or change this field to `Available` or `Active` of state bool.